downloader.py

```diff
@@ -62,13 +62,19 @@ def download_file(filename, prefix, num_videos=None):
         print(e)
 
     num_downloaded = 0
+    available_files = os.listdir('{}_videos'.format(prefix))
```

Comment:
Super minor, but it might be an idea to make available_files a set for faster reads. I think it's also more idiomatic.